test/test_images/hellovolume/hellovolume.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Can you add a service.yaml similar to the ones in https://github.com/knative/serving/pull/3029 ?

Reply:
Done